app/assets/javascripts/discourse/app/components/backup-codes.js

```diff
@@ -42,6 +42,23 @@ export default Component.extend({
     return backupCodes.join("\n").trim();
   },
 
+  @discourseComputed()
+  siteTitleLowerCase() {
```

Comment:
I think having a unit test here will be great as documentation of what this function does :)

Reply:
Have to agree here, especially `^[\000-\177]`, I have no idea what that does.